scripts/publish-prerelease.ts

```diff
@@ -22,7 +23,15 @@ async function main() {
 
 	const versionString = await setCanaryVersions()
 	await uploadStaticAssets(versionString)
-	await publish('canary')
+	await publish(releaseTag)
 }
 
-main()
+const args = process.argv.slice(2)
+if (args.length === 0) {
+	throw new Error('Usage: publish-prerelease.ts <releaseTag>')
+}
+const releaseTag = args[0]
+if (releaseTag !== 'canary' && releaseTag !== 'latest') {
+	throw new Error('releaseTag must be "canary" or "latest"')
+}
```

Comment:
won't this fail when you invoke it with `internal`?

Reply:
yes, oops! i blame the robbot. thanks for pointing this out!